src/compas/artists/artist.py

```diff
@@ -45,6 +45,13 @@ def _get_artist_cls(data, **kwargs):
     else:
         Artist.CONTEXT = identify_context()
 
+    if Artist.CONTEXT is None:
+        error_message = 'No context defined.'
+        error_message += '\n\nThis usually means that the script that you are running requires'
+        error_message += '\na CAD environment but it is being ran as a standalone script'
+        error_message += '\n(ie. from the command line or code editor).'
+        raise Exception(error_message)
```

Comment:
defining custom exception would make the formatting of the message perhaps easier/more elegant? for example `NoArtistContextError`

Reply:
good point, like this? https://github.com/compas-dev/compas/pull/1009/commits/0f9b3ff34bf22b80984eeeaea05bb41f5ec01538

the other one is really a `ValueError`, right? Or do you suggest to also create a custom exception for that?